src/api/app/controllers/staging_project/staged_requests_controller.rb

```diff
@@ -1,5 +1,61 @@
 class StagingProject::StagedRequestsController < StagingProjectController
+  before_action :set_staging_workflow, :set_project, :set_xml_hash
```

Comment:
We should limit the before_action to ``only: [:create, :destroy]``, my fault ...